packages/govern/internal/clients/graphql/queries/dao.ts

```diff
@@ -7,14 +7,20 @@ export interface Dao {
   registryEntries: RegistryEntry[]
 }
 
+export interface Executor {
+  executor: Dao
+}
+
 const dao: string = `
     query DAO($name: String!) {
-      dao(name: $name) {
-        id
-        address
-        metadata
-        registryEntries {
-            ...RegistryEntry_registryEntry
+      executors: registryEntries(where: { name: $name }, first: 1) {
```

Comment:
I think this is wrong.. an entry on the registry exists of Queue/Executor pair without multiple executors.